README.md

```diff
@@ -101,6 +97,35 @@ func main() {
 }
 ```
 
+In order to Marshal/Unamrshal Ion values with annotation, we use a Go struct of type []stirng and tag it with
+`ion:",annotation"`. The other field would be what our Ion value can fit in (both in terms of type and size). For instance,
+to Marshal `age::20`, it must be in a struct as below:
+```GO
+  type foo struct {
+    Value   interface{}
+    AnyName []string `ion:",annotations"`
+  }
+  data := foo{20, []string{"age"}}
+  val, err := MarshalText(data)
+  if err != nil {
+     t.Fatal(err)
+  }
```

Comment:
Maybe add the line that would print out the Ion value as a string and the expected result and comment it out so that the reader would have an idea of the expected output.

Reply:
Thanks for the suggestion. Added.